lib/mix/lib/mix/state.ex

```diff
@@ -0,0 +1,79 @@
+defmodule Mix.State do
```

Comment:
This is required to relup a release containing mix. Multiprocess "fix" is merely coincidence.

@ericmj what do you think?